.gitignore

```diff
@@ -13,7 +15,10 @@ __pycache__/
 /tilegeneration/config.yaml
 /print/print-apps/demo/config.yaml
 /mapserver/mapserver.map.tmpl
+/mapserver/osm.map
+/mapserver/private.map
 /mapserver/*.map.tmpl
+/mapserver/*.map.tmplq
```

Comment:
tmplq?

Reply:
Yep, I don't know what generated that, though. But I do have that file.

Reply:
The file mapserver/edit.map.tmplq.mako should be named as mapserver/edit.map.tmpl.mako :-/